be/src/exec/schema_scanner/schema_partitions_meta_scanner.cpp

```diff
@@ -47,6 +47,7 @@ SchemaScanner::ColumnDesc SchemaPartitionsMetaScanner::_s_columns[] = {
         {"DATA_SIZE", TYPE_VARCHAR, sizeof(StringValue), false},
         {"ROW_COUNT", TYPE_BIGINT, sizeof(int64_t), false},
         {"ENABLE_DATACACHE", TYPE_BOOLEAN, sizeof(bool), false},
+        {"DATA_CACHE_SIZE", TYPE_VARCHAR, sizeof(StringValue), false},
```

Comment:
whether or not a good idea to add new columns in the middle.

Reply:
fix it